runhouse/resources/hardware/cluster.py

```diff
@@ -579,10 +579,17 @@ def _use_https(self) -> bool:
 
     @property
     def _use_nginx(self) -> bool:
-        """Use Nginx if the server port is set to the default HTTP (80) or HTTPS (443) port.
+        """Use Nginx if the server port is set to the default HTTP (80/ 8080) or HTTPS (443 / 8443) port.
         Note: Nginx will serve as a reverse proxy, forwarding traffic from the server port to the Runhouse API
         server running on port 32300."""
-        return self.server_port in [self.DEFAULT_HTTP_PORT, self.DEFAULT_HTTPS_PORT]
+        # Support 8080 and 8443 as defaults for running locally (e.g. in a docker container, to avoid port conflicts
+        # on the host)
+        return self.server_port in [
```

Comment:
@dongreenberg wdyt about adding 8080 http and 8443 https as "default" ports for nginx? useful when running locally in docker